Examples/socketio-python/client_logger.py

```diff
@@ -0,0 +1,48 @@
+import random
+import socketio
+import time
+import logging as log
+
+# Standard python client
+sio = socketio.Client()
+SOCKET_SERVER = 'http://127.0.0.1:8080'
+
+# Default events
+@sio.event
+def connect():
+    print("Connected to server")
+
+
+@sio.event
+def connect_error():
+    print("Connection to server failed!")
+
+
+@sio.event
+def disconnect():
+    print("Disconnected from server")
+    
+    
+@sio.on("telemetry")
+def telemetry(data):
+    print(f"TELEM RECEIVED - {data}")
```

Comment:
This question is just out of curiosity, but could this also be:
```python
@sio.event
def telemetry(data):
```

Reply:
Yeah the function name doesnt matter when using on, only with event